docs/05_best-practices/09_deferred_transactions.md

```diff
@@ -10,3 +10,6 @@ As already mentioned, deferred communication will get scheduled later at the pro
 | As of [EOSIO 2.0 RC1](https://github.com/EOSIO/eos/releases/tag/v2.0.0-rc1) deferred transactions are deprecated.
 
 Due to the above described behaviors it is not recommended to use `deferred transactions`.
+
```

Comment:
suggestions:
v1.8.0 --> v1.8
because technically --> since
starting with version --> starting version
\`v1.8.0\` making it --> \`v1.8\`, making it
to be achieved at --> to do it
On top of that --> Moreover